scenarios/kubernetes_e2e.py

```diff
@@ -473,6 +479,9 @@ def create_parser():
     parser.add_argument(
         '--kubeadm', choices=['ci', 'periodic', 'pull'])
     parser.add_argument(
+        '--kubeadm-kubernetes-version', default=None, help='Version of Kubernetes to use while '
+        'testing kubeadm (e.g. latest, latest-1.6, 1.6.4, etc)')
```

Comment:
I'm a little confused. What logic did you want removed from here? Or are you saying that we should add logic to make sure that if someone specifies `--kubeadm-kubernetes-version` and forgets to set `--kubeadm`, we should have an error, but that new logic should go into a presubmit test instead of a runtime check in the scenario?